apps/webClient/src/components/NavButton/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { makeStyles } from "@material-ui/styles";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+
+interface Props {
+  type: string;
+  text: string;
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  aboutUs: {
+    backgroundColor: "white",
+    borderRadius: 0,
+    width: 60,
+    height: 200,
+  },
+  ourWork: {
+    backgroundColor: "transparent",
+    borderRadius: 0,
+    border: "solid 4px white",
+    width: 60,
+    height: 200,
+  },
```

Comment:
Looking at things closer, I think each of these button styles depend on what page the user is looking at.  For now, let's keep as is, and we'll update the class names and add an `isSelected` prop (or similar) to the component